apps/dashboard/config/routes.rb

```diff
@@ -43,6 +43,7 @@
     post "sessions/:id/cancel", to: "sessions#cancel", as: 'cancel_session'
     scope "*token", constraints: { token: /((usr\/[^\/]+)|dev|sys)\/[^\/]+(\/[^\/]+)?/ } do
       resources :session_contexts, only: [:new, :create]
+      post "session_contexts/save_template/:name", to: "session_contexts#template"
```

Comment:
I don't think we want a new route yet. Or at least, I don't think I want it in this pull request. I think saving the template during launch is best for this initial pull request.

Reply:
Okay, I understand. I will stage those commits in another branch.